ledger-core/rms/proto_vnvn.r0.proto

```diff
@@ -165,14 +178,14 @@ message VDelegatedRequestFinished {
     uint32 CallFlags = 21 [(gogoproto.casttype) = "CallFlags"];
     Reference Callee = 22;
 
-    bytes ResultFlags = 23;
+    Binary ResultFlags = 23;
```

Comment:
Flags cant be bytes ...

Reply:
It's not used anywhere so it's kinda stub field without real type. When it'll be implemented - we'll change it's type and etc